src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java

```diff
@@ -1036,14 +1036,18 @@ private boolean isServerInteresting(GerritTriggeredEvent event) {
                      * Gerrit stream events changed to append approval info to
                      * every comment-added event.
                      **/
-                    if (approval.isUpdated()
-                            && approval.getType().equals(
-                                commentAdded.getVerdictCategory())
-                            && (approval.getValue().equals(
-                                commentAdded.getCommentAddedTriggerApprovalValue())
-                            || ("+" + approval.getValue()).equals(
-                                commentAdded.getCommentAddedTriggerApprovalValue()))) {
-                            return true;
+                    GerritServer server = PluginImpl.getServer_(serverName);
+                    if (server.getGerritVersion() != null
+                            && server.getGerritVersion().compareTo("2.13.0") >= 0) {
```

Comment:
You should implement this using real major.minor.micro version checking.
See https://github.com/jenkinsci/gerrit-trigger-plugin/blob/master/src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/version/GerritVersionChecker.java